src/types/analytics.ts

```diff
@@ -1,4 +1,4 @@
-export type AnalyticStats = {
+export type AnalyticStatsDTO = {
```

Comment:
nit: might be good to add a comment on what DTO stands for, and maybe a link to an article or something, for future developers

Reply:
Good idea, but I'll share this in our Slack channel. DTOs will eventually be pervasive in the codebase.